tests/fixtures/stub_server.py

```diff
@@ -0,0 +1,75 @@
+import SimpleHTTPServer
+import BaseHTTPServer
+from json import dumps
+import re
+
+
+class ServerHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+
+    def do_GET(self):
+        requestline = self.requestline
+        json_dict = parse_id(requestline)
+        if json_dict == {}:
+            self.send_response((400, "Bad Request: File ID does not exist"))
+        self.send_response(200)
+        self.send_header('Content-type', 'application/json')
+        self.end_headers()
+        json = dumps(json_dict)
+        self.wfile.write(json)
+
+
+def run():
+
+    server_address = ('localhost', 8000)
+    httpd = BaseHTTPServer.HTTPServer(server_address, ServerHandler)
+    httpd.serve_forever()
+
+
+def parse_id(path):
+    ids = re.findall(r'FI\d*', path)
+    uuids = re.findall(r'\w{8,8}-\w{4,4}-\w{4,4}-\w{4,4}-\w{12,12}', path)
+    if ids:
+        resp = {"hits": [], "pagination": {"pages": 1}}
+        if "FI250134" in ids:
```

Comment:
Probably better to just put this in a large map and append it if the key exists in the map.